consensus/src/epoch_manager.rs

```diff
@@ -1068,9 +1073,24 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         let consensus_config = onchain_consensus_config.unwrap_or_default();
         let execution_config = onchain_execution_config
             .unwrap_or_else(|_| OnChainExecutionConfig::default_if_missing());
-        let onchain_randomness_config = onchain_randomness_config
-            .and_then(OnChainRandomnessConfig::try_from)
-            .unwrap_or_else(|_| OnChainRandomnessConfig::default_if_missing());
+        let onchain_randomness_config_seq_num = onchain_randomness_config_seq_num
+            .unwrap_or_else(|_| RandomnessConfigSeqNum::default_if_missing());
+
+        info!("local_seqnum={}", self.randomness_override_seq_num);
+        info!(
+            "onchain_seqnum={}",
+            onchain_randomness_config_seq_num.seq_num
+        );
+        let onchain_randomness_config =
+            if onchain_randomness_config_seq_num.seq_num < self.randomness_override_seq_num {
+                info!("Randomness emergency stop.");
```

Comment:
Use error! to alert if validator accidentally overrides its local node config?

Reply:
warned instead